docs/Usage.md

```diff
@@ -105,6 +106,9 @@ If **useRequestIdentity** is set to true, the osb-checker will set `X-Broker-API
 
 **testDashboard** advises the checker to verify if an provided DashboardURL works after creating a service instance.
 
+**contextObjectType** sets the checker to use a ContextObject. Possible object types are kubernetes and cloudfoundry. This option is not set, provision request will be send without 
+a context object.
+
```

Comment:
Giving information about the values the checker sets in the context object would be nice, so I as a developer knows what to expect.